templates/nagios_plugin.py

```diff
@@ -21,7 +21,11 @@ def check_snaps_installed():
 
 
 def check_node(node):
-    checks = [{'name': 'MemoryPressure',
+    checks = [{'name': 'Ready',
```

Comment:
```suggestion
    # Note: Keep the Ready check first since all checks will fail when not Ready
    checks = [{'name': 'Ready',
```
Let's leave a note about why `Ready` needs to go first to prevent future maintainers from stumbling into this same issue.